pages/decision-reviews.tsx

```diff
@@ -156,23 +188,23 @@ export async function getServerSideProps({ req, res, locale }) {
 
   const content = await getDecisionReviewsContent().catch((error) => {
     logger.error(error)
-    return { err: '500' }
+    return { err: '500' } as DecisionReviewContent
   })
```

Comment:
The better way to do this is:

``` typescript
const content = await getDecisionReviewsContent().catch((error): DecisionReviewContent => {
  logger.error(error)
  return { err: '500' }
})
```

as it enforces better typechecks.

As implemented in this PR, a strict typecheck isn't being performed, the return value is being *cast* as `DecisionReviewContent` which could lead to future bugs.

Reply:
fixed in 90106802950b140e4bf12afbe8a110835ff55df6